cime_config/testdefs/testlist_cam.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<testlist version="2.0">
+
+<!--              @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@                -->
+<!--              CAM scientifically supported COMPSETS                 -->
+<!--              @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@                -->
```

Comment:
is this true? I always get confused about what the definition of "scientifically-supported" is.

Reply:
Remnant from my copying.  I've changed the comment.